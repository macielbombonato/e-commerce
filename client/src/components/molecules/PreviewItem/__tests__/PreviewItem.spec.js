import createTestContext from 'react-cosmos-test/enzyme';

import fixture from '../PreviewItem.fixture';

const {mount, getWrapper} = createTestContext({fixture});

beforeEach(mount);

describe('#PreviewItem', () =>
  it('renders component', () => {
    expect(getWrapper()).toMatchSnapshot();
  })
);
