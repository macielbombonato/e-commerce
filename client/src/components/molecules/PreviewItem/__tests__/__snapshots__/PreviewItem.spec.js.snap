// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#PreviewItem renders component 1`] = `
<PreviewItem
  product={
    Object {
      "image": "",
      "name": "product",
      "quantity": 4,
    }
  }
>
  <tr>
    <td
      width="20%"
    >
      <CardImg
        className="img-fluid"
        src="http://localhost:1337/images/"
        tag="img"
        width="100%"
      >
        <img
          className="img-fluid card-img"
          src="http://localhost:1337/images/"
          width="100%"
        />
      </CardImg>
    </td>
    <td>
      product
    </td>
    <td>
      4
    </td>
    <td>
      R$ 
      0.00
    </td>
  </tr>
</PreviewItem>
`;
